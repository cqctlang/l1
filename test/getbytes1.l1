dom = mkdom(c32le, mkzas(1024));
p = (char*){dom}0;
s = getbytes(p, 128);
printf("%d %d\n", isstring(s), strlen(s));
