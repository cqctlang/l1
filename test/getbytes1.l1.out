entry: var params, 4 locals, 0 temps
1 128
